### ISSUE TITLE
[Bug]: When try to log in, it says "Invalid log in credentials" 

### ISSUE THREAD

**Issue by theethylene** [StanGirard/quivr#1360]
### What happened?

A bug happened!

**Comment by jm-rizkallah** [comment]
Check in the supabase authentication page if your user exists and is verified

![Screenshot 2023-10-09 153942](https://github.com/StanGirard/quivr/assets/92910078/3ee59b0a-1465-4142-bbeb-9d12f660734e)

**Comment by stefangweichinger** [comment]
@jm-rizkallah Do I have to create the user there manually? I also have issues getting things up ... and in my case there are no users in supabase. Thanks.

**Comment by BabaSarf** [comment]
I have the same issue .i created user in supabase and try to login but with no success. There is no error on docker logs and i don't see any errors on gui.

**Comment by jm-rizkallah** [comment]
@stefangweichinger did you signup ? after signing up you receive a confirmation mail which will validate your user in supabase.
Authentication is based on supabase
![image](https://github.com/StanGirard/quivr/assets/92910078/ba1ebdb1-4a73-447e-909e-232078de09d3)

**Comment by stefangweichinger** [comment]
@jm-rizkallah I don't get that dialogue (in that form).

It looks like this:

![image](https://github.com/StanGirard/quivr/assets/2673904/816c7ebb-c515-4786-ae5b-c608ea2ea9c1)

**Comment by jm-rizkallah** [comment]
@stefangweichinger guess that's another problem its not loading correctly the frontend ? try rebuilding the containers maybe ?

**Comment by stefangweichinger** [comment]
@jm-rizkallah I did that twice yesterday. On 2 servers.
Also after pulling the latest commits from `main`.

**Comment by jm-rizkallah** [comment]
> @jm-rizkallah I did that twice yesterday. On 2 servers. Also after pulling the latest commits from `main`.

hmm thats weird , maybe try with building with  --no-cache ?

**Comment by stefangweichinger** [comment]
@jm-rizkallah I give it a try ...

**Comment by theethylene** [comment]
<img width="450" alt="image" src="https://github.com/StanGirard/quivr/assets/34483076/707a415d-9231-4b0b-85ff-60f9276b9fde">

**Comment by jm-rizkallah** [comment]
@theethylene did you setup the backend and frontend variables ?

NEXT_PUBLIC_BACKEND_URL=
NEXT_PUBLIC_SUPABASE_URL=
NEXT_PUBLIC_SUPABASE_ANON_KEY=

**Comment by stefangweichinger** [comment]
@jm-rizkallah The rebuild didn't change anything. Same behavior.

As mentioned I edited the variables, but maybe I don't understand everything correctly.

afaik NEXT_PUBLIC_BACKEND_URL should point to localhost:5050 in a docker-compose setup.

I have something like:

```
NEXT_PUBLIC_ENV=local
NEXT_PUBLIC_BACKEND_URL=http://localhost:5050
NEXT_PUBLIC_SUPABASE_URL="https://uuymjSOMEKEY.supabase.co"
NEXT_PUBLIC_SUPABASE_ANON_KEY="eyJhbGciOiJIxxxxxxxxxxyyyy"
```

I tried quotes around the keys and URL and without.

Where would that session storage error come from?

How can I make sure that the stack is able to contact the supabase DB via API-calls? I double- and triple-checked the keys etc

thanks

**Comment by panwankun** [comment]
You can register an account on the localhost page and make sure to click the confirmation link in the email. The account created this way can be logged into on the locally hosted page, tested and verified to be effective.

**Comment by stefangweichinger** [comment]
The registratrion does NOT WORK. There is no user created in the database.

**Comment by theethylene** [comment]
Same for me !

**Comment by jm-rizkallah** [comment]
i think the problem comes from the css not loading and leading to the page not rendering and working correctly

![image](https://github.com/StanGirard/quivr/assets/92910078/ee05cc29-2fda-4d0e-b9fa-bc3b3df6743f)

**Comment by Tam66662** [comment]
> @stefangweichinger guess that's another problem its not loading correctly the frontend ? try rebuilding the containers maybe ?

I know this doesn't answer the main question at hand regarding invalid credentials (one that I'm still struggling with), but I ran into the frontend UI formatting problem from a public domain. For localhost, it looked great, but from the public URL, it looked like what you're seeing in this thread. I was able to fix the frontend UI formatting from public URL by doing the following:
1. Edit frontend's .env file, changing this: `NEXT_PUBLIC_FRONTEND_URL=http://my-public-url.com:3000`
2. Stop the running Quivr containers via Docker Desktop
3. Run from terminal window: `docker compose -f docker-compose.yml up --build`
4. When you open the public URL from a browser, now it looks well-formatted just like localhost:3000
